File: Actividad5/Ejercicio1/CarbonFootprintProject/src/model/Car.java
package model;

public class Car implements CarbonFootprint {

    private double fuelConsumption; // litros

    public Car(double fuelConsumption) {
        this.fuelConsumption = fuelConsumption;
    }

    @Override
    public double getCarbonFootprint() {

        if (fuelConsumption < 0) {
            return 0;
        }

        return fuelConsumption * 2.3;
    }

    @Override
    public String toString() {
        return "Car | Fuel: " + fuelConsumption;
    }
}

File: Actividad6/Ejercicio 1/CarbonFootprintPy/model/car.py
from abc import ABC, abstractmethod

from model.CarbonFootprint import CarbonFootprint

class Car(CarbonFootprint):
    def __init__(self, fuel_consumption: float):
        self.fuel_consumption = fuel_consumption if fuel_consumption >= 0 else 0

    def get_carbon_footprint(self) -> float:
        return self.fuel_consumption * 2.3

    def __str__(self):
        return f"Car | Fuel: {self.fuel_consumption} Litros"
    